backend/src/main/java/hanglog/admin/domain/type/AdminType.java

```diff
@@ -0,0 +1,17 @@
+package hanglog.admin.domain.type;
+
+import static hanglog.global.exception.ExceptionCode.NULL_ADMIN_AUTHORITY;
+
+import hanglog.global.exception.AdminException;
+import java.util.Arrays;
+
+public enum AdminType {
+    ADMIN, MASTER;
```

Comment:
저희 이넘 첫 줄 개행 한것도 있고 안한것도 있던데 이것도 클래스니까 개행합니까? (`trip`패키지의 이넘들은 개행되어 있는데 `auth`는 안되어있음)
```suggestion
public enum AdminType {

    ADMIN, 
    MASTER;
```

Reply:
일단 AdminType은 Auth랑 같은 방식을 따르는게 맞는것 같아서 개행을 안했었는데, Trip까지 전부 보니까 Auth가 잘못되있었던것도 같구요..? 둘 모두 개행 하는 쪽으로 수정하였습니다.